code-of-conduct.html

```diff
@@ -26,18 +26,18 @@
          </nav>
         </div>
 
-		<h1 class="welcome-title">Code of Conduct at RDS</h1>
+		<h1 class="welcome-title">Code of conduct at RDS</h1>
 		<p class="coc-header"> We are dedicated to building a positive environment that inculcates open learning and high respect for one another. So we expect our members to maintain a certain decorum.</p>
 		<ul class="coc-list">
-			<li class="coc-list-elements">No trolling members on the group. We are here to learn and help each other grow. </li>
-			<li class="coc-list-elements">No spamming off topic stuff. Time is a the most important resource. Lets save it in manifolds by reducing off topic content. </li>
+			<li class="coc-list-elements">No trolling of members on the group. We are here to learn and help each other grow. </li>
+			<li class="coc-list-elements">No spamming off topic stuff. Time is the most important resource. Lets save it in manifolds by reducing off topic content. </li>
 			<li class="coc-list-elements">If you've just joined the community, kindly fill out the form to join our discord channel. </li>
 			<li class="coc-list-elements">Avoid Direct Messages. Do not hesitate to ask your queries on the #support discord channel. Aids in quicker response. </li>
 			<li class="coc-list-elements">Avoid using tags @everyone and @here until it is very important.</li>
-			<li class="coc-list-elements">Google your queries. Before your seek help. </li>
-			<li class="coc-list-elements">When seeking help, Submit the code in an easy-to-evaluate fashion. Share a <a href="https://codepen.io/" target="_blank" rel="nopener norefer">Codepen</a> instead of a <a href="https://github.com/" target="_blank" rel="nopener norefer">Github </a>.</li>
-			<li class="coc-list-elements">Overall challenge yourself. Take up tasks poractively and update your progress</li>
-			<li class="coc-list-elements">It helps to have a picture on your discord & Github profiles.</li>
+			<li class="coc-list-elements">Google your queries before you seek help. </li>
+			<li class="coc-list-elements">When seeking help, submit the code in an easy-to-evaluate fashion. Share a <a href="https://codepen.io/" target="_blank" rel="nopener norefer">Codepen</a> instead of a <a href="https://github.com/" target="_blank" rel="nopener norefer">Github </a>.</li>
```

Comment:
Could be: Share a Codepen **link** instead of Github **link**.

Reply:
Sure!